### DISCUSSION TITLE
GCN on Webkb (Texas) dataset

### DISCUSSION THREAD

**Discussion by qubit0** [pyg-team/pytorch_geometric#4898] [Q&A]
I am trying to follow this paper: https://openreview.net/pdf?id=S1e2agrFvS and just try GCN on WebKB (Texas) dataset. I followed what they have in section 4.2 paragraph # 3. But I am getting the mean of 67.03% compared to 52.16% reported in the paper (Table: 3, second row, ninth column)

Here's my implementation: https://colab.research.google.com/drive/1qjW_Rr1zy-KeSPnAdRIlcIgZxtNiRJLf?usp=sharing

Can someone please let me know what am I missing?

**Comment by rusty1s** [comment]
I think your implementation looks correct :)

**Reply by qubit0** [reply]
So, why I am getting much better results than the paper? This very simple GCN is as good as their Geom-GCN, which isn't what they found in the paper. I find this weird.

**Reply by rusty1s** [reply]
I don‘t have a good explanation for this. Maybe you can run their setup as well and try to spot the differences? Maybe it is just a matter of hyperparameter tuning.
